pre_run.py

```diff
@@ -2,19 +2,10 @@
 import os
 import datetime
 from dateutil.relativedelta import relativedelta
+import definitions
 
-
-default_settings = {
-    "included_ticker_lists": ["dividend_aristocrats", "dividaat_list", "challengers", "contenders", "champions"],
-    "start_date": datetime.datetime.strftime(datetime.datetime.now() - relativedelta(years=10), '%Y-%m-%d'),
-    "end_date": datetime.datetime.strftime(datetime.date.today(), '%Y-%m-%d'),
-    "dividend_yield_min_val": 0.020,
-    "payout_ratio_max_val": 0.7,
-    "debt_return_time_max_val_by_ebitda": 5.0,
-    "debt_return_time_max_val_by_income": 5.0,
-    "years_dividends_growth": 5,
-}
-        
+defs = definitions.Definitions()
```

Comment:
move to class __init__

Reply:
I dont think this is the correct approach with this variable.